Content.Client/Lobby/UI/LoadoutPreferenceSelector.xaml.cs

```diff
@@ -24,8 +30,30 @@ namespace Content.Client.Lobby.UI;
 [GenerateTypedNameReferences]
 public sealed partial class LoadoutPreferenceSelector : Control
 {
+    public const string DefaultLoadoutInfoGuidebook = "LoadoutInfo";
+
+    public EntityUid DummyEntityUid;
+    private readonly IEntityManager _entityManager;
+
     public LoadoutPrototype Loadout { get; }
 
+    private LoadoutPreference _preference = null!;
+    public LoadoutPreference Preference
+    {
+        get => _preference;
+        set
+        {
+            _preference = value;
+            NameEdit.Text = value.CustomName ?? "";
+            DescriptionEdit.TextRope = new Rope.Leaf(value.CustomDescription ?? "");
+            ColorEdit.Color = Color.FromHex(value.CustomColorTint, Color.White);
+            if (value.CustomColorTint != null)
+                UpdatePaint(new(DummyEntityUid, _entityManager.GetComponent<PaintedComponent>(DummyEntityUid)), _entityManager);
```

Comment:
So I was planning on asking if you had a way of painting individual layers on clothing items, but now I understand why you wanted the Spraypaint PR. Guh. I don't blame you, Appearance code is awful to work with.